scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -72,8 +70,8 @@
                           name: "{{ custTrack.name }}",
                           type: "{{ custTrack.type }}",
                           format: "{{ custTrack.format }}",
-                          url: "{{cors_anywhere_prefix}}/{{custTrack.url}}",
```

Comment:
Tested this and it's causing this other error:

![image](https://user-images.githubusercontent.com/28093618/106738323-12777900-6618-11eb-82b5-e8ee09eecfdf.png)

Because the dbVar track can't be reached without the right CORS settings either. I think you should comment out the whole custTrack loop block until we find a solution.

Reply:
Yes, foreseen, and sounds good. I was actually looking at proxying the remote track ourselves, via your link, but in a meeting now for a couple of h. 😊